src/main/java/com/cburch/gray/GrayCounter.java

```diff
@@ -48,8 +48,16 @@
  * features beyond the SimpleGrayCounter class.
  */
 class GrayCounter extends InstanceFactory {
+  /**
+   * Unique identifier of the tool, used as reference in project files.
+   * Do NOT change as it will prevent project files from loading.
+   *
+   * Identifier value must MUST be unique string among all tools.
+   */
+  public static final String _ID = "Gray Counter";
```

Comment:
What is your reasoning for adding the underscore in front of `ID`? Other member variables don't use such a prefix. Are you fearing it is to short and could be easily shadowed or shadow something else?

Reply:
Right now, for consistency a with #648 as `_ID` landed there already. I do not think there can be a conflict of any sort and I can have all the `_ID` replaced with just `ID` in next PR.